mvn/archetypes/radix/src/main/resources/archetype-resources/workflow/src/main/resources/etc/workflow.properties

```diff
@@ -54,7 +54,7 @@ org.apache.oodt.cas.workflow.instanceRep.lucene.idxPath=[OODT_HOME]/data/workflo
 # org.apache.oodt.cas.workflow.instanceRep.datasource.quoteFields=false
 
 # XML workflow repository properties
-org.apache.oodt.cas.workflow.repo.dirs=file://[WORKFLOW_HOME]/policy
+org.apache.oodt.cas.workflow.repo.dirs=file:[WORKFLOW_HOME]/policy
```

Comment:
Is this change ok? Did you check after this change?

Reply:
Hi @IMS94,

Same as I mentioned above, I left these intact but hoping to address these inconsistencies in a separate PR.